rest_framework_simplejwt/authentication.py

```diff
@@ -47,7 +47,10 @@ def get_header(self, request):
         Extracts the header containing the JSON web token from the given
         request.
         """
-        header = request.META.get('HTTP_AUTHORIZATION')
+        if api_settings.AUTH_USES_X_ACCESS_TOKEN:
+            header = request.META.get('HTTP_X_ACCESS_TOKEN')
```

Comment:
Again, if we just allow any header the developer specifies, then we can avoid the if statement